proposed/extended-coding-style-guide.md

```diff
@@ -83,7 +83,7 @@ each word is capitalized including the very first letter.
 
 All PHP files MUST use the Unix LF (linefeed) line ending only.
 
-All PHP files MUST end with a non-blank line, terminated with a single LF.
+All PHP files MUST be terminated with a single LF.
```

Comment:
I think this should read

> All PHP files MUST be terminated with a single blank line.

As "blank line" is used throughout the document, and the line ending was defined in the statement above.